docs/source/orderer/ordering_service.md

```diff
@@ -74,41 +74,45 @@ process with three phases that ensures all of the peers in a blockchain network
 keep their ledgers consistent with each other.
 
 In the first phase, a client application sends a transaction proposal to
-a subset of peers that will invoke a smart contract to produce a proposed
-ledger update and then endorse the results. The endorsing peers do not apply
-the proposed update to their copy of the ledger at this time. Instead, the
-endorsing peers return a proposal response to the client application. The
+the Fabric Gateway service. The target peer --- a peer in the same organization
+submitting the proposal --- then runs the transaction. The gateway also forwards
```

Comment:
Just to clarify, the target peer is the one that the client has connected to in order to use the gateway service.  The first endorsement is done by a peer in the target peer's organization.  It could be the target peer itself, but not necessarily (e.g. the target peer might not have the chaincode installed on it).  Also the client might not necessarily be a member of this same organization (e.g. the client's org might not run any peers), but the client *trusts* the target peer's org to handle the transaction.

Reply:
Also, we've lost an important sentence in this change:

"The endorsing peers do not apply the proposed update to their copy of the ledger at this time."

In this phase, the peers are saying what they *would* do with this transaction proposal, given the current state of their individual ledgers.  Only in phase 3, *if and only if all the endorsing peers agree* (i.e. all endorsements match), do the actual changes get applied.

Reply:
yes ok

Reply:
ok